src/wildcard-service/init.lua

```diff
@@ -6,4 +11,67 @@ local mt = {
   __index = _M
 }
 
+function _M.new(options)
+  local opts = options or {}
+  local backend = cache_backend.new(opts.client or resty_backend)
+
+  local http_client = http_ng.new({
+    backend = backend,
+    options = { ssl = { verify = resty_env.enabled('OPENSSL_VERIFY') or false } }
+  })
+
+  local api_host = opts.api_host or getenv('API_HOST') or 'system-provider:3000'
```

Comment:
Guess the default should be `http://system-provider:3000` or it just should do `assert(opts.api)host or getenv('API_HOST'), 'need API_HOST')` to crash when not provided. Because the template will set this anyway the default might not have that much value.